lib/presentation/widgets/sentiment_details.dart

```diff
@@ -99,7 +99,7 @@ class _SentimentCard extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Container(
-      height: 500,
+      height: 455,
```

Comment:
Please revert this change. It causing a bottom overflow

Reply:
Addressed in commit 03882b2c199e1b66065f7db3e2d6295b41e04137